deployments/env_setup/kube-cluster/ansible.cfg

```diff
@@ -0,0 +1,490 @@
+# config file for ansible -- https://ansible.com/
+# ===============================================
+
+# nearly all parameters can be overridden in ansible-playbook
+# or with command line flags. ansible will read ANSIBLE_CONFIG,
+# ansible.cfg in the current working directory, .ansible.cfg in
+# the home directory or /etc/ansible/ansible.cfg, whichever it
+# finds first
+
```

Comment:
@yanmo96 Does this file require? or is this file created by the ansible package installation?

Reply:
This file won't be generate by default when instal ansible. 
I put this file here in case for large scale deployment, so we can adjust how many machines we can do concurrently to speed up the process. (by default is 6, but we change change to 50 or more)